client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -80,7 +80,7 @@ Undocumented.prototype.jetpackModules = function( siteId, fn ) {
  */
 Undocumented.prototype.jetpackModulesActivate = function( siteId, moduleSlug, fn ) {
 	debug( '/sites/:site_id:/jetpack/modules/:module_slug query' );
-	this.wpcom.req.post( { path: '/sites/' + siteId + '/jetpack/modules/' + moduleSlug }, {}, { active: true }, fn );
+	return this.wpcom.req.post( { path: '/sites/' + siteId + '/jetpack/modules/' + moduleSlug }, {}, { active: true }, fn );
```

Comment:
when the only parameter is `path` we can directly pass a string:

```es6
return this.wpcom.req.post( '/sites/' + siteId + '/jetpack/modules/' + moduleSlug, {}, ...
```

Anyway, I know it's out of the purpose of this PR.